terraform/modules/cloud/aws/deploy/fleet/variables.tf

```diff
@@ -0,0 +1,25 @@
+variable "static_nodes" {
+  default = 1
+}
+
+variable "spot_nodes" {
+  default = 9
+}
+
+variable "color" {}
+
+variable "env" {}
+
+variable "instance_type" {
+  default = "t2.micro"
+}
+
+variable "spot_price" {
+  default = "1"
```

Comment:
Can you explain what this would mean?

Reply:
top price that we wish to pay for spot intances, if overall demand will increate to level that spot will cost more then $1 our instances will be terminated

Reply:
OK, so I think the parameters are confusing. I know the `instance_type` is changed by the main module, but I'd like to have instance_type/price at the same file, next to each other.

So, there for instance we should have reasonable spot price for a t2.micro e.g. 0.0116 (that should be the highest on-demand price for that instance (any region?).

Also in `terraform/modules/cloud/aws/deploy/variable.tf` I would like to have `spot_price` as well (to much better clarity) and it should relate to m4.large (the one used in that file), so it should be `0.125`.

For even better clarity the spot price and instance type shall be set in the `main` file that configures the fleets.

In general, I'm against the defaults at all, both in programming and configuration files, it only adds a confusion and second source of truth.

If you're OK, please change. Still, this is not functional issue, so it's up to you. (however the max spot price must be changed to much lower if you choose to keep it here.)